Automation_Custom_Script.sh

```diff
@@ -34,8 +39,12 @@ done
 # Configure passwords and keys
 G_EXEC /boot/dietpi/dietpi-services restart grafana-server
 GLOBAL_PASSWORD=$(openssl enc -d -a -md sha256 -aes-256-cbc -iter 10000 -salt -pass pass:'DietPiRocks!' -in /var/lib/dietpi/dietpi-software/.GLOBAL_PW.bin)
+SERVICE_ACCOUNT_ID=$(\
+  curl --retry 10 --retry-delay 5 --retry-connrefused -s -X POST -H "Content-Type: application/json" -d '{"name":"telegraf", "role": "Admin"}' "http://admin:$GLOBAL_PASSWORD@localhost:3001/api/serviceaccounts" | \
+  python3 -c "import sys, json; print(json.load(sys.stdin)['id'])" \
+)
 GRAFANA_API_KEY=$(\
-  curl --retry 10 --retry-delay 5 --retry-connrefused -s -X POST -H "Content-Type: application/json" -d '{"name":"sailtrack", "role": "Admin"}' "http://admin:$GLOBAL_PASSWORD@localhost:3001/api/auth/keys" | \
+  curl --retry 10 --retry-delay 5 --retry-connrefused -s -X POST -H "Content-Type: application/json" -d '{"name":"telegraf-token"}' "http://admin:$GLOBAL_PASSWORD@localhost:3001/api/serviceaccounts/$SERVICE_ACCOUNT_ID/tokens" | \
```

Comment:
Based on my previous comment, this can simply become "Telegraf":
```suggestion
  curl --retry 10 --retry-delay 5 --retry-connrefused -s -X POST -H "Content-Type: application/json" -d '{"name":"Telegraf"}' "http://admin:$GLOBAL_PASSWORD@localhost:3001/api/serviceaccounts/$SERVICE_ACCOUNT_ID/tokens" | \
```